src/phy.c

```diff
@@ -338,3 +338,176 @@ static int handle_ed_scan(struct nl802154_state *state,
 
 COMMAND(get, ed_scan, "[<page> [<channels> [<duration>]]]",
     NL802154_CMD_ED_SCAN_REQ, 0, CIB_PHY, handle_ed_scan, NULL);
+
+
+static int print_nl802154_beacon_notify_ind(struct nl_msg *msg, void *arg)
```

Comment:
As long as it matches the naming convention that the rest of the callbacks use (e.g. see phy.c).